tests/integration/components/item-meta-test.js

```diff
@@ -0,0 +1,26 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Component | item-meta', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function(assert) {
+    // Set any properties with this.set('myProperty', 'value');
+    // Handle any actions with this.set('myAction', function(val) { ... });
+
+    await render(hbs`<item-meta />`);
```

Comment:
you are going to want to use `{{...}}` invocation here, I think the only reason these tests are passing is because its literally created a DOM element of type `item-meta` and not interpreting it as a component.